### PR TITLE
Fix/facebook lowres

### PR DESCRIPTION
**Pull request by call-me-matt**
as facebook depreciated their API, here is a try to retrieve the public available thumb. Quality is bad, reliability too, still better than nothing. Moved to last priority when fetching different sources.

fixes partly #1891

### CODE REVIEW

**Review by skjnldsv — COMMENTED** [review]
Like mentioned, please use the meta tag with the incorporated image, that is reliable as this is what is used for sharing context creation :)
It feels less hacky and resolution is somewhat decent (200x200)

It works for people that set their visibility to public.
For others, there is nothing we can do :/

**Comment by skjnldsv** [comment]
Also, labels please! :see_no_evil:

File: lib/Service/Social/FacebookProvider.php [hunk 1]
```diff
@@ -59,25 +59,54 @@ public function supportsContact(array $contact):bool {
...
+	 * @param {string} profile url
+	 *
+	 * @return string|null
+	 */
+	protected function getImageUrl(string $profileUrl):?string {
+		try {
+			$result = $this->httpClient->get($profileUrl);
+			$htmlResult = $result->getBody();
+
+			$avatar = '/meta property="og:image" content="(http[^"]+)/';
```
Comment (call-me-matt) [inline lib/Service/Social/FacebookProvider.php:85]: the layout changed and I cannot figure out how to get the picture. but it seems still to be available without login.

**Comment by ChristophWurst** [comment]
I'm closing this due to inactivity. @call-me-matt if this is still a reasonable fix for the facebook images then feel free to reopen and update your branch so it can get a round of review and testing.

**Pull request closed without merging**